content/about/index.md

```diff
@@ -41,3 +41,15 @@ PyBaMM is a Sponsored Project of NumFOCUS, a 501(c)(3) nonprofit charity in the
 <p>
 &nbsp;
 {{< pybamm-twitter >}}
```

Comment:
Should we have the Twitter feed at the end of the page?

Reply:
We can use the shortcode I had added to include it at an upper location if we want, let's wait for a consensus for an apt location

Reply:
Resolving this for now, we can update this later